src/query/plan/operator.hpp

```diff
@@ -2199,6 +2199,8 @@ class CallProcedure : public memgraph::query::plan::LogicalOperator {
   Expression *memory_limit_{nullptr};
   size_t memory_scale_{1024U};
   bool is_write_;
+  mutable utils::MonotonicBufferResource monotonic_memory{1024UL * 1024UL};
+  utils::MemoryResource *memory_resource = &monotonic_memory;
```

Comment:
Why use a pointer if the object is a member of the class?

Reply:
You are right I think, I will check this